dcos/mesos.py

```diff
@@ -567,7 +584,9 @@ def _framework_dicts(self, inactive=False, completed=False):
                 yield framework
 
         for framework in self.state()['frameworks']:
-            if inactive or framework['active']:
+            if framework['active'] and active:
```

Comment:
'active' is a property in the framework dict, that is either True or False. `active` the parameter in the function here, is also a boolean, which means we want to include frameworks that are in the active state. 

I'm not sure what two conditions you are referring to that 'could fit in one line'.